setup.py

```diff
@@ -250,7 +250,7 @@
         'semantic-version>=2.8.5',
         'prettyprinter>=0.18.0',
         'pyroute2>=0.5.14, <0.6.1',
-        'requests>=2.25.0, <=2.31.0',
+        'requests>=2.25.0, <=2.32.3',
```

Comment:
@mlok-nokia what is need of this change?

Reply:
While running "python3 setup.py test" in a docker , recently, it installs request-2.32.2-0y3-none-any.whl.  It requires to make this change to satisfy the condition.  Otherwise, it fails to run the test.  The following is the Error output:
```
Installed /sonic/src/sonic-utilities/.eggs/ordered_set-4.1.0-py3.11.egg
Searching for requests<3.0,>=2.30.0
Reading https://pypi.org/simple/requests/
Downloading https://files.pythonhosted.org/packages/f9/9b/335f9764261e915ed497fcdeb11df5dfd6f7bf257d4a6a2a686d80da4d54/requests-2.32.3-py3-none-any.whl#sha256=70761cfe03c773ceb22aa2f671b4757976145175cdfca038c02654d061d6dcc6
Best match: requests 2.32.3
Processing requests-2.32.3-py3-none-any.whl
Installing requests-2.32.3-py3-none-any.whl to /sonic/src/sonic-utilities/.eggs

Installed /sonic/src/sonic-utilities/.eggs/requests-2.32.3-py3.11.egg
running egg_info
writing sonic_utilities.egg-info/PKG-INFO
writing dependency_links to sonic_utilities.egg-info/dependency_links.txt
writing entry points to sonic_utilities.egg-info/entry_points.txt
writing requirements to sonic_utilities.egg-info/requires.txt
writing top-level names to sonic_utilities.egg-info/top_level.txt
reading manifest template 'MANIFEST.in'
warning: no files found matching '*' under directory 'data'
adding license file 'LICENSE'
writing manifest file 'sonic_utilities.egg-info/SOURCES.txt'
running build_ext
Traceback (most recent call last):
  File "/sonic/src/sonic-utilities/setup.py", line 37, in <module>
    setup(
  File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 108, in setup
    return distutils.core.setup(**attrs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/setuptools/_distutils/core.py", line 185, in setup
    return run_commands(dist)
           ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/setuptools/_distutils/core.py", line 201, in run_commands
    dist.run_commands()
  File "/usr/lib/python3/dist-packages/setuptools/_distutils/dist.py", line 969, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 1213, in run_command
    super().run_command(command)
  File "/usr/lib/python3/dist-packages/setuptools/_distutils/dist.py", line 988, in run_command
    cmd_obj.run()
  File "/usr/local/lib/python3.11/dist-packages/ptr.py", line 208, in run
    with self.project_on_sys_path():
  File "/usr/lib/python3.11/contextlib.py", line 137, in __enter__
    return next(self.gen)
           ^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/setuptools/command/test.py", line 153, in project_on_sys_path
    require('%s==%s' % (ei_cmd.egg_name, ei_cmd.egg_version))
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 956, in require
    needed = self.resolve(parse_requirements(requirements))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 815, in resolve
    dist = self._resolve_dist(
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 861, in _resolve_dist
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (requests 2.32.3 (/sonic/src/sonic-utilities/.eggs/requests-2.32.3-py3.11.egg), Requirement.parse('requests<=2.31.0,>=2.25.0'), {'sonic-utilities'})
```

Reply:
please create separate PR for this change. Why this issue is not seen in PR checker ? is their some gap in our infrastructure ?

Reply:
Agree.  I will remove the setup.py change.  Will create a separated PR if it is needed.